openmdao/core/problem.py

```diff
@@ -47,7 +66,7 @@ class Problem(System):
 
     """
 
-    def __init__(self, root=None, driver=None, impl=None):
+    def __init__(self, root=None, driver=None, impl=None, comm=None):
         super(Problem, self).__init__()
```

Comment:
This will ultimately be required for sub-problems, but the current changes are just enough to be able to run multiple openmdao problems in parallel.  I've added a test (test_mpi.test_multiple_problems)